staging/operator-lifecycle-manager/pkg/controller/bundle/bundle_unpacker.go

```diff
@@ -864,7 +876,7 @@ func sortUnpackJobs(jobs []*batchv1.Job, maxRetainedJobs int) (latest *batchv1.J
 	}
 
 	// cleanup old failed jobs, n-1 recent jobs and the oldest job
-	for i := 0; i < maxRetainedJobs && i+maxRetainedJobs < len(jobs); i++ {
+	for i := 0; i < maxRetainedJobs && i+maxRetainedJobs < len(jobs)-1; i++ {
```

Comment:
Hi @ankitathomas, one concern here, if we use the `< len(jobs)-1` here, it never returns the last child, which means will retain 6 jobs instead of 5. Is it as expected? Thanks! 
```console
maxRetainedJobs := 5                                  // TODO: make this configurable
```

Reply:
That's a good question! The intention there is to preserve the oldest failing unpack job for debugging purposes when this issue happens again, even when it allows for retrying the unpack.

Reply:
Aha, I see now, thanks!